src/Data/List/Relation/Binary/Infix/Homogeneous/Properties.agda

```diff
@@ -8,16 +8,16 @@
 
 module Data.List.Relation.Binary.Infix.Homogeneous.Properties where
 
-open import Level
+open import Data.List.Relation.Binary.Pointwise as Pointwise using (Pointwise)
+open import Data.List.Relation.Binary.Infix.Heterogeneous using (Infix)
+open import Data.List.Relation.Binary.Infix.Heterogeneous.Properties
+  using (fromPointwise; infix?; trans; antisym)
 open import Function.Base using (_∘′_)
+open import Level using (Level; _⊔_)
 open import Relation.Binary.Core using (REL)
-open import Relation.Binary.Structures
+open import Relation.Binary.Structures 
```

Comment:
This will break the whitespace check on commit.
Use `fix-whitespace src/Data/List/Relation/Binary/Infix/Homogeneous/Properties.agda` to fix this.